backend/src/bundles/users/user.service.ts

```diff
@@ -16,8 +16,8 @@ class UserService implements Service {
         this.userRepository = userRepository;
     }
 
-    public find(): ReturnType<Service['find']> {
-        return Promise.resolve(null);
+    public async find(userId: string): Promise<UserEntity | null> {
```

Comment:
There is no need to specify `userId`. If you are working inside of the users bundle it is understandable that find uses user id. REST API principles